### PR TITLE
Few fixes for dependencies

### PR DESCRIPTION
**Pull request by sba1**
This PR contains few fixes that were necessary for compiling Kodi and some of the dependencies on my Debian stretch installation.

### CODE REVIEW

File: tools/depends/target/rapidjson/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-include ../../Makefile.include
+-include ../../Makefile.include
```
Comment (FernetMenta) [inline tools/depends/target/rapidjson/Makefile]: thank you very much. this bothered me too.

**Comment by wsnipex** [comment]
thanks, looks good.
jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by hudokkow** [comment]
This now fails if we're not using unified deps:
> CMake Error at cmake/modules/FindRapidJSON.cmake:22 (find_package):
  Could not find a package configuration file provided by "RapidJSON"
  (requested version 1.1.0) with any of the following names:

>    RapidJSONConfig.cmake
    rapidjson-config.cmake

>  Add the installation prefix of "RapidJSON" to CMAKE_PREFIX_PATH or set
  "RapidJSON_DIR" to a directory containing one of the above files.  If
  "RapidJSON" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  cmake/scripts/common/Macros.cmake:334 (find_package)
  CMakeLists.txt:169 (core_require_dep)

> rapidjson-dev:
  Installed: 1.0.2-2~zesty
  Candidate: 1.0.2-2~zesty
  Version table:
 *** 1.0.2-2~zesty 500
        500 http://ppa.launchpad.net/team-xbmc/xbmc-ppa-build-depends/ubuntu zesty/main amd64 Packages
        500 http://ppa.launchpad.net/team-xbmc/xbmc-ppa-build-depends/ubuntu zesty/main i386 Packages
        100 /var/lib/dpkg/status
     0.12~git20141031+dfsg-1 500
        500 http://pt.archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
        500 http://pt.archive.ubuntu.com/ubuntu zesty/universe i386 Packages

**Comment by wsnipex** [comment]
see https://github.com/xbmc/xbmc/pull/12159

**Comment by sba1** [comment]
Thanks for applying and for the fix of my fix

**Pull request merged**